mailpoet/assets/js/src/automation/editor/components/panel/step-name.tsx

```diff
@@ -0,0 +1,52 @@
+import { Dropdown, TextControl } from '@wordpress/components';
+import { edit, Icon } from '@wordpress/icons';
+import { dispatch, useSelect } from '@wordpress/data';
+import { PlainBodyTitle } from './plain-body-title';
+import { TitleActionButton } from './title-action-button';
+import { store } from '../../store';
+import { Step } from '../workflow/types';
+
+type Props = {
+  step: Step;
+};
+export function StepName({ step }: Props): JSX.Element {
+  const { stepType } = useSelect(
+    (select) => ({
+      stepType: select(store).getStepType(step.key),
+    }),
+    [],
+  );
+  return (
+    <Dropdown
+      className="mailpoet-step-name-dropdown"
+      position="bottom center"
```

Comment:
This positioning doesn't seem to be working for me to reproduce what I'm seeing in Figma. I see a big gap between the action button and the form.

## Figma
<img width="311" alt="image" src="https://user-images.githubusercontent.com/8656640/184237090-c491340e-da79-417a-91da-9215afea6359.png">

## My local firefox
<img width="291" alt="image" src="https://user-images.githubusercontent.com/8656640/184237128-0785888f-870f-4758-9bdc-e05702e2dcfe.png">

## My local Safari
<img width="305" alt="image" src="https://user-images.githubusercontent.com/8656640/184237157-c1f9cf3f-bbe1-4e9f-9490-d3e307f7b295.png">